osquery/logger/plugins/aws_sqs.cpp

```diff
@@ -0,0 +1,118 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <algorithm>
+
+#include <aws/core/client/AWSError.h>
+#include <aws/core/utils/Outcome.h>
+#include <aws/sqs/model/SendMessageBatchRequest.h>
+#include <aws/sqs/model/SendMessageBatchResult.h>
+
+#include <boost/algorithm/string/join.hpp>
+
+#include <osquery/flags.h>
+#include <osquery/registry.h>
+#include <osquery/system.h>
+
+#include "osquery/logger/plugins/aws_sqs.h"
+
+namespace osquery {
+
+REGISTER(SQSLoggerPlugin, "logger", "aws_sqs");
+
+FLAG(uint64,
+     aws_sqs_period,
+     10,
+     "Seconds between flushing logs to SQS (default 10)");
+
+FLAG(string, aws_sqs_queue_url, "", "Name of SQS queue URL for logging")
+
+Status SQSLoggerPlugin::setUp() {
+  initAwsSdk();
+  forwarder_ = std::make_shared<SQSLogForwarder>(
+      "aws_sqs", aws_sqs_queue_url, 500);
+  Status s = forwarder_->setUp();
+  if (!s.ok()) {
+    LOG(ERROR) << "Error initializing SQS logger: " << s.getMessage();
+    return s;
+  }
+  Dispatcher::addService(forwarder_);
+  return Status(0, "OK");
```

Comment:
@fmanco What is the decided on style for new Status OK objects now?

Reply:
Status::success(msg)

Reply:
it looks like msg isn't actually required here and `Status::success()` will suffice?